client/my-sites/email/email-management/home/email-mailbox-action-menu.jsx

```diff
@@ -326,15 +326,76 @@ RemoveTitanMailboxConfirmationDialog.propTypes = {
 	setVisible: PropTypes.func.isRequired,
 };
 
-const EmailMailboxActionMenu = ( { account, domain, mailbox } ) => {
+const EmailForwardRemovalResponseMessage = ( { mailbox, successful } ) => {
 	const dispatch = useDispatch();
 	const translate = useTranslate();
+
+	const emailAddress = getEmailAddress( mailbox );
+
+	const noticeDuration = 7000;
+
+	const errorMessage = errorNotice(
+		translate(
+			'There was an error removing {{strong}}%(emailAddress)s{{/strong}} from your account',
```

Comment:
But users can only have mailboxes *or* email forwards for each domain right? 

We could modify this string, but since we don't explicitly mention the word "mailbox", I thought it was neat that translators wouldn't have to handle a new string (this string is the same as when Titan mailboxes are removed).

Reply:
> But users can only have mailboxes or email forwards for each domain right?

It's rare, but maintaining the exclusivity depends on the client code. 

Now when I think of it, it's rare enough that it shouldn't make a difference.